src/main/webapp/index.jsp

```diff
@@ -37,8 +37,8 @@
     <div class="col-xs-12 col-sm-8">
       <h2 class="media-heading">Praised by Users:</h2>
       <p id="testimonialContainer">
-        Thank you for building such a wonderful tool.
-        <br>-Faculty user, Canada
+        I just started exploring TEAMMATES and am very impressed. Wish I discovered it earlier.
+        <br>Faculty user, Singapore
```

Comment:
Use `<br>-Faculty user, Singapore` or `<br>- Faculty user, Singapore`. This applies to the strings in the .es6 file as well.

If using the `<br>- Faculty user, Singapore` variation (with a space between the dash and the next word), then update all strings in the .es6 file to match this convention.